e2e/flows/envs.e2e.3.js

```diff
@@ -65,8 +66,8 @@ describe('envs', function () {
     helper.addFileToEnvInBitJson(undefined, '.babelrc', './.babelrc', COMPILER_ENV_TYPE);
     helper.addToRawConfigOfEnvInBitJson(undefined, 'a', 'b', COMPILER_ENV_TYPE);
     helper.addToRawConfigOfEnvInBitJson(undefined, 'valToDynamic', 'valToDynamic', COMPILER_ENV_TYPE);
-    helper.createFile('', 'mocha-config.js', '{"someConfKey": "someConfVal"}');
-    helper.addFileToEnvInBitJson(undefined, 'config', './mocha-config.js', TESTER_ENV_TYPE);
+    helper.createFile('', 'mocha-config.opts', '{"someConfKey": "someConfVal"}');
```

Comment:
Current (in development) mocha tester in envs doesn't support opts. Don't see why we should store it.  it can only be on the this path `/test/mocha.opts`. Right now the configuration will be in bit.json

Reply:
In this PR, I only changed the extension from .js to .opts so it won't be parsed mistakenly as a JS file.
Why was it done originally? I guess for just a quick example of adding a configuration file to Mocha.

Reply:
@davidfirst  that's correct.. I didn't use this file for real.. only check it was written and read by bit